frontend/src/features/book/BookListSection.tsx

```diff
@@ -0,0 +1,80 @@
+import RefetchButton from "@/components/RefetchButton";
+import Tabs from "@/components/Tabs";
+import type { BookType } from "@/features/book/Book.types";
+import BookCard from "@/features/book/BookCard";
+import { bookDataList } from "@/mocks/bookMocks";
+import { theme } from "@/styles/themes.style";
+import { css } from "@emotion/react";
+import type { Dispatch, SetStateAction } from "react";
+const { color, typography } = theme;
+
+interface BookListSectionProps {
+  TAB_LIST: BookType[];
+  activeTab: BookType;
+  setActiveTab: Dispatch<SetStateAction<BookType>>;
+}
+const BookListSection = ({
+  TAB_LIST,
+  activeTab,
+  setActiveTab,
+}: BookListSectionProps) => {
+  const LOCATION_SECTION = "운정 1구역";
+  return (
+    <div css={BookListSectionContainer}>
+      <div css={HeaderContainer}>
+        <p css={LocationSectionText}>{LOCATION_SECTION}</p>
+        <RefetchButton handleClick={() => {}} />
+      </div>
+      <Tabs
+        type="bar_true"
+        group={TAB_LIST}
+        selected={activeTab}
+        setSelected={setActiveTab}
+      />
+      <div css={ContentContainer}>
+        {bookDataList.map((bookData, idx) => (
+          <div key={bookData.name}>
+            <BookCard bookType={activeTab} data={bookData} />
+            {idx !== bookDataList.length - 1 && <div css={LineWrapper} />}
+          </div>
```

Comment:
맨 아래 요소 제외하고 밑줄 보여주는 목적이면 아래 코드도 좋을 것 같아요 !
```suggestion
       {bookDataList.map((bookData, idx) => (
          <div key={bookData.name} css={BookCardWrapper}>
            <BookCard bookType={activeTab} data={bookData} />
          </div>
```
아래에 스타일 추가:
```ts
const BookCardWrapper = css`
  padding-bottom: 20px;
  &:not(:last-of-type) {
    border-bottom: 1px solid ${color.GrayScale.gray3};
  }
`;
```